src/Hl7.Fhir.Shims.STU3AndUp/Specification/Terminology/ValueSetExpander.cs

```diff
@@ -0,0 +1,348 @@
+﻿#nullable enable
```

Comment:
This is a dead file, it is not part of the project and is not compiled. No review necessary (it's a copy of the other ValueSetExpander.cs anywat).